### PR TITLE
Documentation and readme reorganisation

### PR DESCRIPTION
**Pull request by ewels**
1. New pipeline logo
2. Review and restructure of the results documentation
3. Separation of the main readme into new files in the `docs` folder.

### CODE REVIEW

**Comment by Hammarn** [comment]
Nice! I'll make some comments. Saw some small formatting error and did a re-read and found some stuff that has been outdate.

File: docs/usage.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+### `--saveReference`
+Supply this parameter to save any generated reference genome files to your results folder.
+These can then be used for future pipeline runs, reducing processing times.
+
+## Job Resources
+### Automatic resubmission
+Each step in the pipeline has a default set of requirements for number of CPUs,
+memory and time. For most of the steps in the pipeline, if the job exits
+on UPPMAX with an error code of `143` (exceeded requested resources) it will
+automatically resubmit with higher requests (2*original, then 3*original).
```
Comment (Hammarn) [inline docs/usage.md]: This gives italics... better to use x instead of *

File: docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+Submit arbitrary SLURM options (UPPMAX profile only). For instance, you could use `--clusterOptions '-p devcore'`
+to run on the development node (though won't work with default process time requests).
+
+### `-c`
+Specify the path to a specific config file (this is a core NextFlow command). Useful if using different UPPMAX
+projects or different sets of reference genomes. **NB:** one hyphen only (core Nextflow parameter).
+
+## Stand-alone scripts
+There is a folder with some unmaintained standalone scripts that you can use:
+[stand-alone scripts](https://github.com/SciLifeLab/NGI-RNAseq/blob/master/stand-alone-scripts).
```
Comment (Hammarn) [inline docs/usage.md]: This link is broken and the paragraph is outdated. Maybe we should mention the bin directory were the executed scripts are stored and that there are standalone stuff in there?

File: docs/usage.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+If you prefer, you can use [HISAT2](https://ccb.jhu.edu/software/hisat2/index.shtml) as the
+alignment tool instead. Thought of as the successor to Tophat by many, HISAT2 has a much smaller
+memory footprint.
+
+To use HISAT2, use the parameter `--aligner hisat2` or set `params.aligner = 'hisat2'` in your config file.
+
+## Reference Genomes
+
+### `--genome`
+The reference genome to use of the analysis, needs to be one of the genome specified in the config file.
```
Comment (Hammarn) [inline docs/usage.md]: > use **of** the analysis
shouldn'tit be **for**?

File: docs/output.md [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+* [StringTie](#stringtie) - FPKMs for genes and transcripts
+* [Sample_correlation](#Sample_correlation) - create MDS plot and sample pairwise distance heatmap / dendrogram
+* [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
+
+## FastQC
+[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your reads. It provides information about the quality score distribution across your reads, the per base sequence content (%T/A/G/C). You get information about adapter contamination and other overrepresented sequences.
+
+For further reading and documentation see the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
+
+> **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality. To see how your reads look after trimming, look at the FastQC reports in the `trim_galore` directory.
```
Comment (Hammarn) [inline docs/output.md]: Oh this reminds me, I think we still don't actually include these in the results 😊

File: docs/output.md [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+
+**Output directory: `results/fastqc`**
+
+* `sample_fastqc.html`
+  * FastQC report, containing quality metrics for your untrimmed raw fastq files
+* `sample_fastqc.zip`
+  * zip file containing the FastQC report, tab-delimited data file and plot images
```
Comment (Hammarn) [inline docs/output.md]: Are these supposed to be lists? They don't show up like that for me. Maybe it works on github 😕 . Same thing for virtually all the other programs paragraphs.

File: docs/output.md [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+> _Credit: [dupRadar documentation](https://www.bioconductor.org/packages/devel/bioc/vignettes/dupRadar/inst/doc/dupRadar.html)_
+
+**Output directory: `results/dupRadar`**
+
+* `Sample_markDups.bam_duprateExpDens.pdf`
+* `Sample_markDups.bam_duprateExpBoxplot.pdf`
+* `Sample_markDups.bam_expressionHist.pdf`
+* `Sample_markDups.bam_dupMatrix.txt`
+* `Sample_markDups.bam_duprateExpDensCurve.txt`
+* `Sample_markDups.bam_intercept_slope.txt`
```
Comment (Hammarn) [inline docs/output.md]: These are correctly displayed as a list for me

File: docs/output.md (hunk 4 above)
Comment (ewels) [inline docs/output.md]: We do now!

**Comment by Hammarn** [comment]
Looking sharp! Merging!
image

**Pull request merged**